### ISSUE TITLE
It doesn't obfuscate delegate name

### ISSUE THREAD

**Issue by lextudio-support** [obfuscar/obfuscar#532]
like this one:
```
[UnmanagedFunctionPointer(CallingConvention.Cdecl)]
public delegate void __Delegate_DeleteSimulator(IntPtr handler);
```
this is my config file:
```
<?xml version='1.0'?>
<Obfuscator>
	<Var name="InPath" value="D:\download\dota\test_Data\Managed\org" />
	<Var name="OutPath" value="D:\download\dota\test_Data\Managed\" />
	<Var name="LogFile" value="$(OutPath)\mapping.log"/>
	<Var name="KeepPublicApi" value="false"/>
	<Var name="HidePrivateApi" value="true"/>
	<Var name="UseUnicodeNames" value="false"/>
	<Var name="UseKoreanNames" value="false"/>
	<Var name="ReuseNames" value="false"/>
	<Var name="HideStrings" value="false"/>
	<Module file="$(InPath)\Assembly-CSharp.dll">
		<SkipType rx=".*" skipMethods="true" skipFields="true"/>
		<ForceType name="ChatApi"/>
		<ForceType name="SimulationApi"/>
	</Module>
</Obfuscator>
```

**Original Reported Date**: 2014-10-30T20:36:36.8-07:00
**Planned For Release**: 
**Original CodePlex ID**: 22

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

ok. It's my fault. Delegate is a type, not a field.
So add <ForceType/> it works.

**Original Posted Date**: 2014-10-30T21:45:45.923-07:00

**Comment by lextudio-support** [comment]
**Original Closed Date**: 2014-10-30T22:56:09.09-07:00
